script/build-example-site.js

```diff
@@ -0,0 +1,22 @@
+var marked = require('marked');
+var fs = require('fs');
+var path = require('path');
+var rimraf = require('rimraf');
+
+var basePath = path.resolve(__dirname, '../') + '/';
+
+try {
+  rimraf.sync(basePath + 'resource/example-site');
+  fs.mkdirSync(basePath + 'resource/example-site');
+} catch(e) {
+  console.log(e);
+}
+
+var template = fs.readFileSync(basePath + 'resource/example-site.template.html', {encoding: 'utf-8'});
+var readmeText = fs.readFileSync(basePath + 'README.md', {encoding: 'utf-8'});
+
+var exampleSiteHtml = template
+  .replace('<!-- REPLACE -->', marked(readmeText))
+  .replace('src="resource', 'src="https://raw.githubusercontent.com/james-proxy/james/master/resource');
+
+fs.writeFileSync(basePath + 'resource/example-site/index.html', exampleSiteHtml);
```

Comment:
Maybe this could be moved to gulp, so that the target (`basePath + target + '/example-site/index.html')` can be dynamic.
The `resources` folder is usually full of constants, I'm not sure that it makes sense to put temporary files in there